interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,6 +35,402 @@ components:
+      example: 78ea3fe8-71c2-4f5c-9b44-9c02f5563c6f
+
+
+    OperationalIntentUssBaseURL:
+      description: >-
+        The base URL of a USS implementation that implements the parts of the
+        USS-USS API necessary for providing the details of this operational
+        intent, and telemetry during non-conformance or contingency, if
+        applicable.
+      anyOf:
+        - $ref: "#/components/schemas/UssBaseURL"
+
+    OperationalIntentState:
+      description: >-
+        State of an operational intent.
+        'Accepted': Operational intent is created and shared, but not yet in
+        use; see standard text for more details.
+        The create or update request for this operational intent reference must
+        include a Key containing all OVNs for all relevant Entities.
+        'Activated': Operational intent is in active use; see standard text for
+        more details.
+        The create or update request for this operational intent reference must
+        include a Key containing all OVNs for all relevant Entities.
+        'Nonconforming': UA is temporarily outside its volumes, but the
+        situation is expected to be recoverable; see standard text for more
+        details.
+        In this state, the `/uss/v1/operational_intents/{entityid}/telemetry`
+        USS-USS endpoint should respond, if available, to queries from USS
+        peers.  The create or update request for this operational intent may
+        omit a Key in this case because the operational intent is being adjusted
+        as flown and cannot necessarily deconflict.
+        'Contingent': UA is considered unrecoverably unable to conform with its
+        coordinate operational intent; see standard text for more details.
+        This state must transition to Ended.  In this state, the
+        `/uss/v1/operational_intents/{entityid}/telemetry` USS-USS endpoint
+        should respond, if available, to queries from USS peers.  The create or
+        update request for this operational intent may omit a Key in this case
+        because the operational intent is being adjusted as flown and cannot
+        necessarily deconflict.
+      type: string
+      enum:
+      - Accepted
+      - Activated
+      - Nonconforming
+      - Contingent
+    Key:
+      description: >-
+        Proof that a client has obtained the latest airspace content, used to
+        ensure that write operations to the DSS occur only when the latest
+        content is known (i.e. has been read). The client is required to pass a
+        current Key constructed from information obtained with previous read
+        operations and subsequent requests for full information, and optionally,
+        subscription updates for deconflicted write operations to the DSS.  The
+        contents of this data structure are generated by the client.
+      type: array
+      items:
+        $ref: '#/components/schemas/EntityOVN'
+      default: [ ]
+    SubscriptionUssBaseURL:
```

Comment:
Since ImplicitSubscriptionParameters is not used in this interface. this schema could be removed.

Reply:
Thank you for this, removing.